camel-idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin url="https://github.com/camel-idea-plugin/camel-idea-plugin">
   <id>org.apache.camel</id>
   <name>Apache Camel IDEA Plugin</name>
-  <version>0.6.2</version>
+  <version>0.6.5</version>
```

Comment:
Need to be changed to v0.6.4

Reply:
I figured that but then I saw some change logs on 0.6.4. I'll set it to 0.6.4